src/js/stores/draganddrop.js

```diff
@@ -169,7 +207,7 @@ define(function (require, exports, module) {
                 }.bind(this));
             }
 
-            if ((potentialDropTarget && potentialDropTarget.validate(this._dragTarget, point, potentialDropTarget.b))) {
+            if ((potentialDropTarget && potentialDropTarget.validate(this._dragTarget, point, this._currentBounds))) {
```

Comment:
This is a fairly hot function as a mousemove handler, and I can imagine a number of ways to speed it up:

1. Sort `_dropTargets` (say, by their y-coordinate) and search for matches using a binary instead of a linear search. Currently this is an O(n^2) algorithm, since you're doing an O(n) lookup on O(n) drop targets.
2. Memoize `potentialDropTarget.validate`.
3. Possibly consider caching the result of `getBoundingRect` since it's so expensive. Could we demand that the bounding rects of potential _drop targets_ must not change during the drag operation?
4. This is minor, but you should move the search procedure to a private method so you can avoid the binding step. (This would probably be insignificant in most cases, but perhaps not in a mousemove handler.)